### ISSUE TITLE
[FEATURE REQUEST] Compatibility mode to avoid conflicts in the overview

### ISSUE THREAD

**Issue by G-dH** [micheleg/dash-to-dock#2043]
Hello,
I'm developing an extension ([V-Shell](https://github.com/G-dH/vertical-workspaces)) that needs control over overview allocations and startup animation that DtD also overrides. It would be nice if DtD offered an option to disable injections into related functions. V-Shell supports DtD using ugly workarounds, but conflicts are inevitable and sometimes fatal, especially at startup. Even if the DtD changes its position, the V-Shell must respond with its own reset to regain control of the allocations. All of this degrades the user experience, adds work to me and of course from the user's point of view it's all my fault, because DtD is "just a dock".

**Comment by bmonjoie** [comment]
Please consider this feature request. V-Shell is an amazing extension, and it would greatly improve the user experience of both your users.

**Comment by Aqua1ung** [comment]
I second this: Please work with @G-dH to fix this conflict!

**Comment by DelayedHappiness** [comment]
Agreed. I've been experiencing issues with the compatibility between them both. On login it makes it so that the display stays on a white screen!
